tensorflow_recommenders_addons/dynamic_embedding/core/kernels/lookup_impl/lookup_table_op_gpu.h

```diff
@@ -166,6 +173,10 @@ class TableWrapper final : public TableWrapperBase<K, V> {
                 << ", load factor=" << std::setprecision(2)
                 << (float)total_size / (float)new_capacity << "].";
     }
+    CUDA_CHECK(cudaStreamSynchronize(stream));
+    new_hint_size = table_->get_size(stream);
```

Comment:
The `get_size` makes the code confusing. It would be better to call `cudaDeviceSynchronize` directly in insert & accum.

Reply:
fix